tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     pytest
     pytest-capturelog
     pytest-cov
+    dictdiffer
```

Comment:
Is this necessary? I think it will be installed along with the other things in requirements.txt.